styles.css

```diff
@@ -1,44 +1,35 @@
-body{
-    background-color:whitesmoke;
-
-}
-h1{
-    text-align: center;
+body {
+  background-color: whitesmoke;
+  width: 100%;
 }
-.image{
-    background-color: white;
-    
-    
-
-
 
+h1 {
+  text-align: center;
 }
-img{
-    padding: 50px;
-    margin-left: 60px;
-    width: 80%;
-    height:20%;
 
-
-}
-figcaption{
-    padding-bottom: 20px;
-    padding-left: 80px;
-    font-style: italic;
-    
+.image {
+  background-color: white;
+  padding-bottom: 20px;
+  padding: 80px;
+  font-style: italic;
 }
 
-.world{
-    margin-left: 10px;
-    padding-left: 350px;
+img {
+  padding: 50px;
+  width: 80%;
+  height: 20%;
+}
 
 
+.world {
+  margin: 2px;
+  padding: 5px;
 }
-h3.world{
 
-    text-align: center;
+h3.world {
```

Comment:
- [ ] I think the student is right too, doing h3.class and .class h3 are 2 different things, for the first action, the code is applied to an h3 element with class `world` whereas h3 .world applies to any elements with class `world` that are within tag element; which is same to `.world h3`, why do you think using your syntax is preferable than the students? please explain the why to the student

Reply:
- [ ] And again, all pull request with failed linters is marked as invalid, in your case you give the student a review, I understand because you wanted us to see the review comment but, please use a different bug to tag, check the code or whatever but not the linter, please
- [ ] Make at least to comments in addition to the one above